.changeset/gentle-cobras-kick.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/plugin-search-backend-module-elasticsearch': patch
+---
+
+Feature: add a new option to set the batch size for elastic search engine, if not given the default batch size is 1.000
+
+Example usage:
```

Comment:
Thanks for referencing this example here.  Would you mind also adding an example to documentation?  Could be useful to highlight _why_ one might need to do this (e.g. if you are using lower spec compute resources, like what you ran into).

Under `Other` here could be a good place: https://github.com/backstage/backstage/blob/master/docs/features/search/search-engines.md#others

Reply:
@iamEAP Thanks for your feedback, I've improved the documentation, can you please review it? :) Thanks!